source/language/gates.rst

```diff
@@ -132,7 +132,15 @@ From a physical perspective, the gates :math:`e^{i\gamma}U` and :math:`U` are eq
 phase :math:`e^{i\gamma}`. When we add a control to these gates, however, the global phase becomes a relative phase
 that is applied when the control qubit is one. To capture the programmer's intent, a built-in global phase gate
 allows the inclusion of arbitrary global phases on circuits. The instruction ``gphase(γ);`` adds a global phase
-of :math:`e^{i\gamma}` to the scope containing the instruction. For example
+of :math:`e^{i\gamma}` to the gate, subroutine, or global scope which containing the instruction.
```

Comment:
I'm not sure if it was ever spelled out, but I believe `gphase` was only intended to appear in a gate definition. Maybe @blakejohnson can clarify

Reply:
cc @ajavadia and @levbishop, since Blake's on holiday.

Reply:
In my mind `gphase` can appear anywhere. However, it's only visible if it gets `ctrl` or `negctrl` modified. Thus, it can make a difference if used in a `gate` body, since `gates` can become controlled. Subroutines and global scopes can't be control modified so the global phase isn't observable from those contexts.

An inline-controlled `gphase` such as `ctrl @ gphase(theta) q;` is not actually applying a global phase and is observable in any scope.